src/ctia/bundle/core.clj

```diff
@@ -442,9 +470,23 @@
    identity-map
    ident
    params
-   services :- APIHandlerServices]
+   {{:keys [send-event]} :RiemannService
+    :as services} :- APIHandlerServices]
   (if (seq ids)
-    (->> (map #(export-entities % identity-map ident params services) ids)
-         (reduce #(deep-merge-with coll/add-colls %1 %2))
-         (into empty-bundle))
+    (binding [correlation-id (str (java.util.UUID/randomUUID))]
+      (send-event {:service "export-bundle"
+                   :correlation-id correlation-id
+                   :time (get-epoch-second)
+                   :event-type "start"
+                   :metric (count ids)})
```

Comment:
The `service` field should be different for each event type. The metric is directly related to the service. We cannot mix the number of ids and the processing time of the `export-bundle` fn in the metric field if the service is the same. Here, we could rename the service as `Export bundle`. In the [Riemann documentation](http://riemann.io/concepts.html), they use `API port 8000 reqs/sec` as an example.

```suggestion
      (send-event {:service "Export bundle start"
                   :correlation-id correlation-id
                   :time (get-epoch-second)
                   :event-type "export-bundle"
                   :metric (count ids)})
```

You can keep a field like `event-type` to group all events related to the export bundle.

Nothing is wrong but it's more related to keep the same approach for all our Riemann events

Reply:
You're right, will change it